test/test-udp-mmsg.c

```diff
@@ -82,12 +83,18 @@ static void recv_cb(uv_udp_t* handle,
     return;
   }
 
-  ASSERT_EQ(nread, 4);
-  ASSERT_NOT_NULL(addr);
-  ASSERT_MEM_EQ("PING", rcvbuf->base, nread);
+  if (nread == 0) {
+    /* There can be no more available data for the time being. */
+    ASSERT_NULL(addr);
+  } else {
+    ASSERT_EQ(nread, 4);
+    ASSERT_NOT_NULL(addr);
+    ASSERT_MEM_EQ("PING", rcvbuf->base, nread);
+    received_datagrams++;
+  }
 
   recv_cb_called++;
-  if (recv_cb_called == NUM_SENDS) {
+  if (received_datagrams == NUM_SENDS) {
     uv_close((uv_handle_t*)handle, close_cb);
     uv_close((uv_handle_t*)&sender, close_cb);
```

Comment:
style nit (while you are here)
```suggestion
    uv_close((uv_handle_t*) handle, close_cb);
    uv_close((uv_handle_t*) &sender, close_cb);
```

Could you also fix the alignment of the function arguments?

Reply:
done